index.js

```diff
@@ -1,159 +1,98 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "extends": "stylelint-config-standard",
-    "customSyntax": "scss",
-    "plugins": [
-        "stylelint-scss"
-    ],
-    "ignoreFiles": [
-        "sass/vendor/*.scss"
-    ],
-    "rules": {
-        "at-rule-empty-line-before": [
-            "always",
+    'extends': 'stylelint-config-standard-scss',
+    'customSyntax': 'postcss-scss',
+    'rules': {
+        'alpha-value-notation': 'number',
+        'at-rule-empty-line-before': [
+            'always',
             {
-                "except": [
-                    "blockless-after-same-name-blockless",
-                    "first-nested"
-                ],
-                "ignore": [
-                    "after-comment"
-                ],
-                "ignoreAtRules": [
-                    "extend",
-                    "include",
-                    "else",
-                    "elseif",
-                    "content"
+                except: ['blockless-after-blockless', 'first-nested'],
+                ignore: ['after-comment'],
+                ignoreAtRules: [
+                    'media',
+                    'return',
+                    'extend',
+                    'include',
+                    'else',
                 ]
             }
         ],
-        "at-rule-no-unknown": [
-            true,
+        'block-no-empty': true,
+        'color-hex-length': 'long',
+        'comment-no-empty': true,
+        'declaration-empty-line-before': 'never',
```

Comment:
@mscharl Our preference is to not have spaces between declarations. I would like to keep it like this, if it's okay for you? It's easy to overwrite this rule.

Reply:
@thomasduenser It's okay for me. We'll most probably overwrite this for our projects.
Out of curiosity, why do you prefer no empty lines? Or is it just out of habit?

Reply:
@mscharl Thanks 👍🏻

It's a combination of the following reasons:
- we try to avoid classes with too many properties
- not every developer would "group" the same properties together and use the empty lines in the right places, resulting in a mixed codebase
- and yes, also because we're used to it